src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,132 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
+//import MicroscopyMetadataTool from "4dn-microscopy-metadata-tool";
```

Comment:
One of the dependencies used (html2canvas) in micrometa depends on `window`, hence we only import the component via code-splitting after componentDidMount (=guaranteed to be client-side).

We would have done this anyway in the future since we don't need to bundle micrometa on initial load of homepage, for example.